src/Illuminate/Mail/Transport/SesTransport.php

```diff
@@ -57,17 +57,21 @@ protected function doSend(SentMessage $message): void
         }
 
         try {
-            $result = $this->ses->sendRawEmail(
+            $result = $this->ses->sendEmail(
                 array_merge(
                     $options, [
-                        'Source' => $message->getEnvelope()->getSender()->toString(),
-                        'Destinations' => collect($message->getEnvelope()->getRecipients())
-                                ->map
-                                ->toString()
-                                ->values()
-                                ->all(),
-                        'RawMessage' => [
-                            'Data' => $message->toString(),
+                        'ReplyToAddresses' => [$message->getEnvelope()->getSender()->toString()],
+                        'Destination' => [
+                            'ToAddresses' => collect($message->getEnvelope()->getRecipients())
```

Comment:
What if they are `cc` addresses?

Reply:
If there is a way to decide between `to`, `bcc` and `cc` from the `SentMessage`, I would love to implement it:
[[SentMessage]()](https://docs.aws.amazon.com/aws-sdk-php/v3/api/api-sesv2-2019-09-27.html#sendemail)

Reply:
Have you tested sending emails with cc and bcc recipients with this PR in place? What is the behavior?

Reply:
@taylorotwell I tested `bcc` and `cc` and they work as expected.